include/d/actor/d_a_ship.h

```diff
@@ -238,8 +241,8 @@ class daShip_c : public fopAc_ac_c {
     /* 0x040C */ f32 m040C;
     /* 0x0410 */ int mGridId;
     /* 0x0414 */ daGrid_c* mpGrid;
-    /* 0x0418 */ int m0418;
-    /* 0x041C */ fopAc_ac_c* m041C;
+    /* 0x0418 */ int mTornadoID;
```

Comment:
```suggestion
    /* 0x0418 */ u32 mTornadoID;
```
Proc IDs normally use the `fpc_ProcID` type (aka uint), but based on onTornadoFlg taking a u32 as its parameter this one may be typed as `u32`. Either way, I don't think `int` is correct, that's just a guess we had in Ghidra because this field was unknown, you might need to change these initial guesses to match the debug maps.